### PR TITLE
Make service ipFamilies configurable

### PR DESCRIPTION
**Pull request by poksiala**
This pull request is mostly an reference to the linked issue, but I can develop it further if it is something you would like to merge. 

Current Helm chart does not work in dual-stack cluster. This configuration addition makes it possible to force services into IPv4 only mode.

See: https://github.com/longhorn/longhorn/issues/2452

### CODE REVIEW

**Comment by jenting** [comment]
Thank you for your contribution @poksiala
I'll review and test it this week.

File: chart/templates/deployment-ui.yaml [hunk 1]
```diff
@@ -48,6 +48,13 @@ spec:
   {{- else }}
   type: {{ .Values.service.ui.type }}
   {{- end }}
+  type: {{ .Values.service.ui.type }}
```
Comment (jenting) [inline chart/templates/deployment-ui.yaml:51]: Is this line duplicated?

**Review by jenting — COMMENTED** [review]
@poksiala comment left

File: chart/templates/deployment-ui.yaml (hunk 1 above)
Comment (jenting) [reply chart/templates/deployment-ui.yaml:51]: @poksiala would you mind taking a look at the comment I left?

Comment (poksiala) [reply chart/templates/deployment-ui.yaml:51]: Hey sorry I did not get a notification about your earlier message. I'll fix this a it later today.

Comment (jenting) [reply chart/templates/deployment-ui.yaml:51]: Many thanks @poksiala

**Comment by jenting** [comment]
Thank you @poksiala. Would you mind add DCO to the latest commit?

**Review by jenting — CHANGES_REQUESTED** [review]
Thank you @poksiala. Would you mind add DCO to the latest commit?

**Comment by poksiala** [comment]
> Thank you @poksiala. Would you mind add DCO to the latest commit?

Do you mean signing the commit? I can do it in few hours once I get to my PC

**Comment by jenting** [comment]
> > Thank you @poksiala. Would you mind add DCO to the latest commit?
> 
> Do you mean signing the commit? I can do it in few hours once I get to my PC

Yes, please.

**Comment by poksiala** [comment]
> > > Thank you @poksiala. Would you mind add DCO to the latest commit?
> > 
> > 
> > Do you mean signing the commit? I can do it in few hours once I get to my PC
> 
> Yes, please.

Done!

**Review by jenting — APPROVED** [review]
LGTM, thank you @poksiala

**Comment by shuo-wu** [comment]
Notice that this feature is announced beta version in v1.21. And the prerequisite is v1.20. Is it OK to merge this PR now?
https://kubernetes.io/docs/concepts/services-networking/dual-stack/#prerequisites

**Comment by innobead** [comment]
This PR has been pending for a while. Closing first, and feel free to continue if you want.

**Pull request closed without merging**